frontend/src/hooks/api/geocoder/useApiGeocoder.ts

```diff
@@ -14,6 +20,36 @@ export const useApiGeocoder = () => {
       findAddresses: async (address: string) => {
         return api.get<IGeoAddressModel[]>(`/tools/geocoder/addresses?address=${address}`);
       },
+      addresses: async (address: string, bbox?: string, locality?: string) => {
+        const axios = Axios.create({
+          baseURL: 'https://geocoder.api.gov.bc.ca',
+        });
+        // TODO: Figure out how to only return good results, Geocoder returns a ton of junk.
+        var params: any = {
```

Comment:
I've tried a number of parameters to improve the results, but haven't had much luck.